### ISSUE TITLE
minpac#update() not working on Windows 10 + nvim 0.5.0

### ISSUE THREAD

**Issue by ghillb** [k-takata/minpac#122]
Hello,

oddly `:call minpac#update()` fails for me on Windows 10 and nvim 0.5.0 with the following error:
```
Error detected while processing function <SNR>57_start_update[2]..<SNR>57_update_single_plugin[62]..<SNR>57_start_job[12]..minpac#job#start[1]..<SNR>59_job_start:
line   29:
E903: Process failed to start: no such file or directory: "/bin/bash"
```

OS: Windows 10 `v1809`
nvim version: `NVIM v0.5.0-801-ga061d53e1`
installed via: `scoop install neovim-nightly`
minpac installed via: `git clone https://github.com/k-takata/minpac.git %LOCALAPPDATA%\nvim\pack\minpac\opt\minpac`

**resulf of :call minpac#update():**
<img width="458" alt="Screenshot 2020-11-03 204206" src="https://user-images.githubusercontent.com/35503959/98033359-ab2e0b00-1e15-11eb-9922-626dff14b98f.png">

On Linux minpac works flawlessly for me!

**Comment by k-takata** [comment]
I haven't used Neovim, so I'm not sure why this occurs.
What does `:set shell?` show? I expect `shell=C:\Windows\System32\cmd.exe`.
Do you set `SHELL` environment variable?

**Comment by ghillb** [comment]
You are absolutely right, the shell was pointing to bash. After removing it, `:call minpac#update()` worked fine.

**Comment by k-takata** [comment]
Can you copy&paste the result of `:message`?

**Comment by ghillb** [comment]
It is unfortunately just what you see in the screenshot:
```
Error detected while processing function <SNR>15_HL:
line   43:
E421: Color name or number not recognized: ctermbg= gui=NONE cterm=NONE
E421: Color name or number not recognized: ctermbg= gui=NONE cterm=NONE
E421: Color name or number not recognized: ctermbg= gui=NONE cterm=NONE
```
